src/plugins/guided_onboarding/public/constants/guides_config/security.ts

```diff
@@ -9,22 +9,20 @@
 import type { GuideConfig } from '../../types';
 
 export const securityConfig: GuideConfig = {
-  title: 'Get started with SIEM',
+  title: 'Elastic Security guided setup',
```

Comment:
Do you mind updating the strings to use `i18n` as part of this PR?

Reply:
yes, I completely forgot about it, thanks a lot!